code/modules/mob/living/carbon/alien/alien.dm

```diff
@@ -38,7 +38,8 @@
 
 	create_internal_organs()
 
-	AddElement(/datum/element/ventcrawling, given_tier = VENTCRAWLER_ALWAYS)
+	if(!(name == "alien queen"))
+		AddElement(/datum/element/ventcrawling, given_tier = VENTCRAWLER_ALWAYS)
```

Comment:
Not a great way to do this, better add a 'can ventcrawl' var to xenos that it checks for before adding the component, and set it to false to ones that shouldn't have it.
Also praetorians should also be incapable of ventcrawling.